docs/REST API/rest-api.md

```diff
@@ -5765,6 +5765,9 @@ This endpoint returns the list of custom MDM commands that have been executed.
 | per_page                  | integer | query | Results per page.                                                         |
 | order_key                 | string  | query | What to order results by. Can be any field listed in the `results` array example below. |
 | order_direction           | string  | query | **Requires `order_key`**. The direction of the order given the order key. Options include `asc` and `desc`. Default is `asc`. |
+| host_identifier           | integer or string  | query | The host's `hardware_serial`, `uuid`, `osquery_host_id`, `hostname`, or `node_key` |
```

Comment:
i believe this is typically just a `string`

Reply:
@mostlikelee That's good point. I'm not sure about `osquery_host_id` and `node_key`. Maybe one of these is an integer? We're going to remove these from the docs so I think it makes sense to change type to `string` only.